src/main/java/Storage.java

```diff
@@ -0,0 +1,41 @@
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class Storage {
+    private static FileWriter fileWriter;
+    private String fileName;
+
+    /**
+     * @param file name of txt file to store data whenever TaskList changes
+     *
+     */
+    public Storage(String file) {
+        fileName = file;
+    }
+
+    /**
+     * writes and saves all Task data into a txt file.
+     * @param taskManager a TaskList object
+     *
+     */
+    public void saveTask(TaskList taskManager) throws IOException {
+        String message = taskManager.getTasksData();
+        fileWriter = new FileWriter(this.fileName);
+        fileWriter.write(message);
+        fileWriter.close();
+    }
+
+    /**
+     * calls saveTask().
+     * @param taskManager a TaskList object
+     *
+     */
+    public void saveTaskUtil(TaskList taskManager) {
+        try {
+            saveTask(taskManager);
+        } catch (IOException e) {
```

Comment:
You should not catch this IOException unless you are prepared to offer a way for the user to recover from it. It should crash the program.